measurement/dash_apps/plots.py

```diff
@@ -25,6 +25,31 @@ def create_empty_plot() -> px.scatter:
     return fig
 
 
+def get_aggregation_level(timeseries: pd.Series, aggregate: bool = False) -> str:
+    """Calculates the aggregation level based on the timeseries separation.
+
+    Args:
+        timeseries: Time data to be aggregated.
+        aggregate: Flag indicating if there should be aggregation
+
+    Return:
+        String indicating the aggregation level as " - LEVEL UNITS aggregation" or an
+        empty string if no aggregation is required.
+    """
+    if not aggregate:
+        return ""
+
+    aggregation = timeseries.diff().dt.seconds.mean() / 60
```

Comment:
I guess this will be thrown off if there are any large gaps in the data, so maybe take the median instead?

Reply:
That's a good point. I'll change it.